test/SlowTests/Issues/RavenDB_19900.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+using System;
+using System.Threading;
+using Raven.Client.Http;
+using Tests.Infrastructure;
+
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_19900 : ClusterTestBase
+    {
+        public RavenDB_19900(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task UpdateTopologyTimeoutBehaviourShouldBeAccordingToConfiguration()
```

Comment:
please figure out how to change this test so it will not take 30+ seconds each time.

Reply:
Done